notebooks/fig1.ipynb

```diff
@@ -0,0 +1,1561 @@
+       "TACGAAGGGTGCAAGCGTTGTTCGGAATAACTGGGCGTAAAGCGCAC...  c__Deltaproteobacteria   \n",
+       "TACGGAGTGTGCAAGCGTTACTCGGAATCACTGGGCATAAAGAGCAC...       c__028H05-P-BN-P5   \n",
+       "TACGAGAGGTCCAAACGTTATTCGGAATTACTGGGCTTAAAGAGTTC...       c__Planctomycetia   \n",
+       "\n",
+       "                                                            taxonomy_3  \\\n",
+       "featureid                                                                \n",
+       "TACGGAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGCAC...  o__Alteromonadales   \n",
+       "TACAGAGGTCCCAAGCGTTGTTCGGATTCATTGGGCGTAAAGGGCTC...         Unspecified   \n",
+       "TACGAAGGGTGCAAGCGTTGTTCGGAATAACTGGGCGTAAAGCGCAC...         Unspecified   \n",
+       "TACGGAGTGTGCAAGCGTTACTCGGAATCACTGGGCATAAAGAGCAC...                 o__   \n",
+       "TACGAGAGGTCCAAACGTTATTCGGAATTACTGGGCTTAAAGAGTTC...     o__Pirellulales   \n",
+       "\n",
+       "                                                             taxonomy_4  \\\n",
+       "featureid                                                                 \n",
+       "TACGGAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGCAC...  f__Alteromonadaceae   \n",
+       "TACAGAGGTCCCAAGCGTTGTTCGGATTCATTGGGCGTAAAGGGCTC...          Unspecified   \n",
+       "TACGAAGGGTGCAAGCGTTGTTCGGAATAACTGGGCGTAAAGCGCAC...          Unspecified   \n",
+       "TACGGAGTGTGCAAGCGTTACTCGGAATCACTGGGCATAAAGAGCAC...                  f__   \n",
+       "TACGAGAGGTCCAAACGTTATTCGGAATTACTGGGCTTAAAGAGTTC...     f__Pirellulaceae   \n",
+       "\n",
+       "                                                            taxonomy_5  \\\n",
+       "featureid                                                                \n",
+       "TACGGAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGCAC...  g__Marinimicrobium   \n",
+       "TACAGAGGTCCCAAGCGTTGTTCGGATTCATTGGGCGTAAAGGGCTC...         Unspecified   \n",
+       "TACGAAGGGTGCAAGCGTTGTTCGGAATAACTGGGCGTAAAGCGCAC...         Unspecified   \n",
+       "TACGGAGTGTGCAAGCGTTACTCGGAATCACTGGGCATAAAGAGCAC...                 g__   \n",
+       "TACGAGAGGTCCAAACGTTATTCGGAATTACTGGGCTTAAAGAGTTC...                 g__   \n",
+       "\n",
+       "                                                     taxonomy_6  \n",
+       "featureid                                                        \n",
+       "TACGGAGGGTGCAAGCGTTAATCGGAATTACTGGGCGTAAAGCGCAC...          s__  \n",
+       "TACAGAGGTCCCAAGCGTTGTTCGGATTCATTGGGCGTAAAGGGCTC...  Unspecified  \n",
+       "TACGAAGGGTGCAAGCGTTGTTCGGAATAACTGGGCGTAAAGCGCAC...  Unspecified  \n",
+       "TACGGAGTGTGCAAGCGTTACTCGGAATCACTGGGCATAAAGAGCAC...          s__  \n",
+       "TACGAGAGGTCCAAACGTTATTCGGAATTACTGGGCTTAAAGAGTTC...          s__  "
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tbl_biom = load_table(\"fig1/input/EMP_tbl.biom\")\n",
```

Comment:
Not needed, but thought I would share: `qiime tools import` can extract taxonomic information from biom files. For example `qiime tools import --input-path t.biom --output-path taxonomy.qza --type FeatureData[Taxonomy] --input-format BIOMV210Format` ... or something like that :)